frontend/src/component/project/Project/PaginatedProjectFeatureToggles/hooks/useDefaultColumnVisibility.test.tsx

```diff
@@ -0,0 +1,74 @@
+import theme from 'themes/theme';
+import { screen } from '@testing-library/react';
+import { useDefaultColumnVisibility } from './useDefaultColumnVisibility';
+import { render } from 'utils/testRenderer';
+import { ThemeProvider } from 'themes/ThemeProvider';
+import mediaQuery from 'css-mediaquery';
+
+function createMatchMedia(width: number) {
+    return (query: string) => {
+        return {
+            matches: mediaQuery.match(query, { width }),
+            media: '',
+            addListener: () => {},
+            removeListener: () => {},
+            onchange: () => {},
+            addEventListener: () => {},
+            removeEventListener: () => {},
+            dispatchEvent: () => true,
+        };
+    };
+}
+
+function resizeScreen(width: number) {
+    window.matchMedia = createMatchMedia(width);
+}
```

Comment:
This bit allows us to resize the screen in the tests (based on [this article](https://www.js-howto.com/test-responsive-design-using-jest-and-react-testing-library/) that I found). I'm wondering whether we should extract it into a separate file or leave it here for now.